src/applications/mhv/secure-messaging/tests/containers/Folders.unit.spec.jsx

```diff
@@ -13,12 +15,28 @@ describe('Folders Landing Page', () => {
     },
   };
 
-  it('renders without errors', () => {
-    const screen = renderWithStoreAndRouter(<Folders />, {
-      initialState,
+  const setup = (state = initialState) => {
+    return renderWithStoreAndRouter(<Folders />, {
+      initialState: state,
       reducers: reducer,
       path: `/folders`,
     });
+  };
+
+  let screen = null;
+  beforeEach(() => {
+    screen = setup();
+  });
+
+  it(`verifies page title tag for 'My folders' page`, async () => {
+    await waitFor(() => {
+      expect(global.document.title).to.equal(
+        PageTitles.MY_FOLDERS_PAGE_TITLE_TAG,
+      );
+    });
+  });
+

```

Comment:
Do you think it is feasible to expand test coverage to other pages beyond a landing page and My Folders?